public/editor/scripts/main.js

```diff
@@ -52,6 +52,16 @@ require(["jquery", "bowser"], function($, bowser) {
 
   Bramble.once("error", onError);
 
+  var errorMessageTimeoutMS = 10000;
+
+  setTimeout(function(){
+    showLoadingErrorMessage();
+  }, errorMessageTimeoutMS);
+
+  function showLoadingErrorMessage(){
+    $("#spinner-container .taking-too-long").addClass("visible");
+  }
+
   $("button.refresh-browser").on("click",function(){
     window.location.reload(true);
```

Comment:
Same thing here, can we add analytics so we get some insight into how often this is happening?